LEAF_Request_Portal/templates/print_subindicators.tpl

```diff
@@ -28,12 +28,12 @@
             <!--{/if}-->
             <div style="float: right">
             <!--{if $date < $indicator.timestamp && $date > 0}-->
-                <img src="../libs/dynicons/?img=appointment.svg&amp;w=16" alt="View History" title="View History" style="cursor: pointer" onclick="getIndicatorLog(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->)" />&nbsp;
+                <img src="../libs/dynicons/?img=appointment.svg&amp;w=16" alt="View History" title="View History" style="cursor: pointer" onclick="getIndicatorLog(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->)" tabindex="0" role="button" onkeypress="if (event.keyCode==13){ getIndicatorLog(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->); }"/>&nbsp;
             <!--{/if}-->
             <!--{if $indicator.isWritable == 0}-->
-                <img src="../libs/dynicons/?img=emblem-readonly.svg&amp;w=16" alt="Read-only" title="Read-only" />
+                <img src="../libs/dynicons/?img=emblem-readonly.svg&amp;w=16" alt="Read-only" title="Read-only" tabindex="0" role="button" />
             <!--{else}-->
-                <img src="../libs/dynicons/?img=accessories-text-editor.svg&amp;w=16" alt="Edit this field" title="Edit this field" style="cursor: pointer" onclick="getForm(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->)" />
+                <img src="../libs/dynicons/?img=accessories-text-editor.svg&amp;w=16" alt="Edit <!--{$indicator.name|sanitizeRichtext|strip_tags}--> field" title="Edit <!--{$indicator.name|sanitizeRichtext|strip_tags}--> field" style="cursor: pointer" onclick="getForm(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->)" tabindex="0" role="button" onkeypress="if (event.keyCode==13){ getForm(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->); }" />
```

Comment:
In these context santizerRichText cannot be used. When used in alt-text, variables should be stripped of all tags.